### PR TITLE
Update User Guide for View, Close commands

### LINKED ISSUE #105: Update UG for View and Close command

### PR DESCRIPTION
**Pull request by iamdiluxedbutcooler**
closes #105

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -202,6 +202,8 @@ Here is a reference table that briefly summarizes available commands:
 | `delete`    | Removes a client from the system.                      |
 | `list`      | Displays all clients currently stored in the system.   |
 | `filter`    | Filters clients based on specified criteria            |
+| `view`      | Opens a split view showing detailed client information |
+| `close`     | Closes the split view of client details               |
```
Comment (itsme-zeix) [inline docs/UserGuide.md:206]: Perhaps we can standardize whether we want to use `client details` or `detailed client information`.

**Review by itsme-zeix — APPROVED** [review]
Great work! Thank you for updating the UG, especially with the formatting fixes/standardizations too!

**Review by FionaQY — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (ZShunRen) [reply docs/UserGuide.md:206]: I would say `detailed client information` is more befitting of this context.

**Review by ZShunRen — APPROVED** [review]
LGTM

**Pull request merged**